docs/platform/release-notes/tizen-5-5-m2.md

```diff
@@ -0,0 +1,641 @@
+  - Layout bugs have been fixed.
+  - Memory leaks have been fixed.
+  - Crashes have been fixed.
+
+
+## Tizen .NET
+
+### New and changed features
+
+- .NET Core (Runtime)
+  - .NETCore has been upgraded from 3.0.0-preview2 to 3.0.0.
+- Xamarin.Forms
+  - Xamarin.Forms 4.2.0, which is latest stable version has been supported.
+- Tizen.CircularUI
+  - Wearable UI extension (Tizen.Wearable.CircularUI) 1.3.0 has been supported.
+- TizenFX (C# Device API) API Level 6
+  - New Application API set for component ID has been added.
+  - New ComponentBased API set has been added.
+  - New AudioManager API set for audio ducking has been added.
+  - New PackageManager API set to get dependency information has been added.
+  - New MachineLearning Inference API set has been added.
+  - New MediaContent API set to add face information into the database manually has been added.
+  - Not used filter keywords of MediaContent has been deprecated.
+  - New AudioChannel and SampleType of AudioIO have been added.
+  - New ImageUtil API set to extract thumbnail has been added.
+  - New MediaPlayer API set to export video frames and audio PCM data has been added.
+  - New MediaPlayer API set to cancel preparing has been added.
+  - New MediaPlayer API set for audio codec and offload has been added.
+  - New Multimedia Remote API set for subtitles and 360 mode has been added.
+  - New Vision API set for detection and classification using the inference engine has been added.
+  - New UI component set of NUI has been added.
+  - Tizen.NUI.UIComponents module has been deprecated.
+  - New NUI API set for window creation and rotation has been added.
+  - New SystemSettings API set to get rotary event enabled has been added.
+  - New VoiceControlManager API set has been added.
+
+### Known Issues
+
+- Xamarin.Forms
+  - For more information on the list of limitations, see [here](https://developer.tizen.org/development/api-reference/.net-application/current-xamarin.forms-limitations).
+
+
+## Toolchain
+
+### New and changed features
+
+- Toolchain watermarking (annobin) function has been added.
+- AddressSanitizer shadow scaling option has been applied to reduce runtime overhead on RAM.
+- libasansi library has been introduced to support dotnet ASan runtime.
+- gcov-force-options package has been added. This package has scripts to enable automatic package coverage instrumentation.
+
+### Fixes
+
+- GCC
+  - Bugfix on Internal Compiler Error (ICE) has been back-ported from upstream - bug #83623 #85496.
+  - LeakSanitizer allocator bug on 64-bit target (aarch64, x86_64) has been fixed.
+  - IntegerSanitizer option's bug with "-fsanitize=shift,unsigned-integer-overflow" has been fixed.
+  - AddressSanitizer runtime option has been modified (allocator_may_return_null=1) to provide better report.
+- GLIBC
+  - fclose SIGABORT issue on multi-thread environment with race condition has been fixed by applying bug [#15142](https://sourceware.org/bugzilla/show_bug.cgi?id=15142).
```

Comment:
applying bug [#15142](https://sourceware.org/bugzilla/show_bug.cgi?id=15142).

How about "applying bug fix of [#15142](https://sourceware.org/bugzilla/show_bug.cgi?id=15142)."?

Reply:
will accept